packages/react-celo/src/use-celo-methods.ts

```diff
@@ -257,15 +257,50 @@ export function useCeloMethods(
 
 export interface CeloMethods {
   resetInitError: () => void;
```

Comment:
@aaronmgdr I just noticed that `resetInitError` and `initConnector` are the only ones without description in the `CeloMethods` interface. Might as well add it!
Would it be correct to say they're `@private`? Also, can you help me write up what they're used for? 🙏

Reply:
When a connector fails to initialize due to some error being throw the `setConnectionInitialise` action is dispatched. that error is then displayed to the user.  `restInitError` is use to clean that up.  

`initConnector`  is used when a user clicks on a connector.

Reply:
But neither of those are exported, so can I mark them as private?

Reply:
sure or internal?